_sass/support/mixins/_typography.scss

```diff
@@ -1,86 +1,98 @@
 // Font size
+$font-size-1: 9px;
```

Comment:
This is a great idea, but we should keep all of our variables in the `variables.scss` file. 

Could you move these there, and also use an `-sm` suffix instead of `-mq` so that it's more specific to the case (small screen size)?